### PR TITLE
Align DS challenge readme and notebook prompt

### PR DESCRIPTION
**Pull request by sjmccorm1993**
Clarify requirements and expectations in data science challenge notebook and align prompt in README.md to match what's in the notebook.

### CODE REVIEW

File: data-science/Data_science_exercise.ipynb [hunk 1]
```diff
@@ -12,18 +12,22 @@
...
     "   errors.\n",
     "2. Feel free to use any libraries you like. Your final results should be\n",
     "   presented in this Python notebook.\n",
     "3. There are no constraints on the techniques you bring to bear, we are curious\n",
     "   to see how you think and what sort of resources you have in your toolbox.\n",
-    "4. Be sure to clearly articulate what you did, why you did it, and how the\n",
-    "   results should be interpreted. In particular you should be aware of the\n",
+    "4. **Be sure to clearly articulate what you did, why you did it, and how the\n",
+    "   results should be interpreted**. In particular, you should be aware of the\n",
     "   limitations of whatever approach or approaches you take.\n",
```
Comment (rjlauer) [inline data-science/Data_science_exercise.ipynb:31]: 4. is less of a "note" but a repeat of objectives 2. and 3. . That said, we know it cannot hurt to overly stress this, so I'm fine either way, leaving or removing it.

**Review by rjlauer — APPROVED** [review]
LGTM!

**Review by kylelrichards11 — APPROVED** [review]
LGTM

**Comment by annahaensch** [comment]
Apparently I'm still getting notifications from this repository, haha.  Anyhow, LGTM, have a nice weekend :)

**Pull request merged**